store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java

```diff
@@ -33,6 +33,9 @@ public class MessageStoreConfig {
     @ImportantField
     private String storePathCommitLog = null;
 
+    @ImportantField
+    private String storePathDLedgerCommitLog = null;
```

Comment:
Why not reuse storePathCommitLog?

Reply:
@Jason918  If we use the path of commitLog directly, there will be incompatibility in the mixed commitLog. That is, the commitLog of Master-slave will be read by DLedger.

Reply:
@Jason918  I found this case by the unit test.
https://github.com/apache/rocketmq/blob/50e314eb8ada23283dcdbb261766183c90fd435c/store/src/test/java/org/apache/rocketmq/store/dledger/MixCommitlogTest.java#L31